base_layer/wallet/src/key_manager_service/storage/sqlite_db/mod.rs

```diff
@@ -47,41 +42,16 @@ const LOG_TARGET: &str = "wallet::key_manager_service::database::wallet";
 #[derive(Clone)]
 pub struct KeyManagerSqliteDatabase {
     database_connection: WalletDbConnection,
-    cipher: Arc<RwLock<XChaCha20Poly1305>>,
 }
 
 impl KeyManagerSqliteDatabase {
     /// Creates a new sql backend from provided wallet db connection
     /// * `cipher` is used to encrypt the sensitive fields in the database, if no cipher is provided, the database will
```

Comment:
These comments don't match the current workings and fields

Reply:
Addressed